### ISSUE TITLE
app crashing to homescreen

### ISSUE THREAD

**Issue by steelersfan7** [jellyfin-archive/jellyfin-roku-legacy#397]
**Describe the bug**
When opening the latest version of roku on a TCL roku tv the app crashes when clicking on a poster

**To Reproduce**

1. select any library on roku
2. see posters
3. click on a poster
4. end up back to the home screen

**Expected behavior**
more consistient behavior when it comes to viewing the metadata for movie/show
**Logs**
I do not know how to get logs but I did try:

downloading eclipse, cloning the repo, installing the plugin and connecting to the roku with telnet. I see just a console. 
 tried running anything from here https://developer.roku.com/docs/developer-program/debugging/debugging-channels.md and nothing happens.

I did come across the link below which states in the developer dashboard it would log crashes:

https://community.roku.com/t5/Roku-Developer-Program/Bright-Script-Crash-Log/m-p/566477

**Screenshots**
no screenshots, click on a library, click on a poster and it ends up back on the main screen.

**Additional context**
this is a TCL roku tv on software version 9.4.2 build 4101-88(latest for me)
this behavior exhibits on stable and beta versions of jellyfin from the roku store

**Comment by anthonylavado** [comment]
@neilsb Could this be that `null` event you were looking into?

**Comment by jimdogx** [comment]
Don't know if this is related but I was using a Roku USB stick with Jellyfin just fine.  Then I recently got a new TCL Roku TV and any time I tried to play anything it would crash back to the Home Screen.  After some debugging I found that this crash was the same one mentioned here: 

https://github.com/jellyfin/jellyfin-roku/issues/467

Changing the one line (mediaSourceId = "") fixed the issue for me.
